src/blocks/time.rs

```diff
@@ -78,44 +97,59 @@ impl ConfigBlock for Time {
     ) -> Result<Self> {
         let i = Uuid::new_v4().to_simple().to_string();
         Ok(Time {
+            time: Local::now(),
             id: i.clone(),
             format: block_config.format,
-            time: ButtonWidget::new(config, i.as_str())
+            format_short: block_config.format_short,
+            widget: ButtonWidget::new(config, i.as_str())
                 .with_text("")
+                .with_short_text("")
                 .with_icon("time"),
             update_interval: block_config.interval,
             on_click: block_config.on_click,
             timezone: block_config.timezone,
+            width: block_config.width,
         })
     }
 }
 
 impl Block for Time {
     fn update(&mut self) -> Result<Option<Update>> {
-        let time = match self.timezone {
-            Some(tz) => Utc::now().with_timezone(&tz).format(&self.format),
-            None => Local::now().format(&self.format),
+        self.time = match self.timezone {
+            Some(tz) => Local.from_utc_datetime(&Utc::now().with_timezone(&tz).naive_local()),
```

Comment:
Unrelated change? If so please open a new PR.

Reply:
Oups, it was a leftover of a first try. I'll push a commit that remove self.time. However I need to store the datetime instead of the formated string as the datetime is then formatted differently for short_text and full_text